Tests/SpiderTest.php

```diff
@@ -4,12 +4,16 @@
 
 use PHPUnit_Framework_TestCase;
 use Simgroep\ConcurrentSpiderBundle\Spider;
+use Simgroep\ConcurrentSpiderBundle\CrawlJob;
 use VDB\Uri\Uri;
 use Symfony\Component\EventDispatcher\GenericEvent;
 
 class SpiderTest extends PHPUnit_Framework_TestCase
 {
-    public function testIfDoubleSlashIsRemoved()
+    /**
+     * @Test
+     */
+    public function isDoubleSlashIsRemoved()
```

Comment:
change name of function to begin with test in name so we have everything a consistency so everything without name in test will be internally only for phpunit